### PR TITLE
Used Get-AzAccessToken cmdlet

### PR DESCRIPTION
**Pull request by stefanstranger**
Replaced code to retrieve Access Token with Get-AzAccessToken cmdlet and replaced code that used that Token variable.

The documentation does not use the (new) Get-AzAccessToken cmdlet. e.g. https://github.com/Azure/Enterprise-Scale/blob/main/docs/Deploy/enable-subscription-creation.md

```PowerShell
# Fetching new token
$azureRmProfile = [Microsoft.Azure.Commands.Common.Authentication.Abstractions.AzureRmProfileProvider]::Instance.Profile
$profileClient = [Microsoft.Azure.Commands.ResourceManager.Common.RMProfileClient]::new($azureRmProfile)
$token = $profileClient.AcquireAccessToken($currentContext.Tenant.Id)
```
Should be changed to:

```PowerShell
$Token = Get-AzAccessToken
```
[Link to issue](https://github.com/Azure/Enterprise-Scale/issues/395)

### CODE REVIEW

File: docs/Deploy/enable-subscription-creation.md [hunk 1]
```diff
@@ -28,9 +28,7 @@ $spnObjectId = ""
 $currentContext = Get-AzContext
```
Comment (hjscherer) [inline docs/Deploy/enable-subscription-creation.md:28]: "$currentContext" can also be removed

**Review by hjscherer — CHANGES_REQUESTED** [review]
"$currentContext" variable also not required anymore.

**Review by krnese — APPROVED** [review]

File: docs/Deploy/enable-subscription-creation.md (hunk 1 above)
Comment (krnese) [reply docs/Deploy/enable-subscription-creation.md:28]: Pls see comment from Hansjoerg

Comment (stefanstranger) [reply docs/Deploy/enable-subscription-creation.md:28]: Yes the variable currentContext is obsolete now.

Comment (hjscherer) [reply docs/Deploy/enable-subscription-creation.md:28]: Awesome thank you @stefanstranger

**Review by hjscherer — APPROVED** [review]

**Pull request merged**